frontend/src/app/collaboration/collaboration.component.html

```diff
@@ -1,8 +1,13 @@
-<p-splitter styleClass="w-full background b1">
+<p-splitter styleClass="w-full background">
     <ng-template pTemplate>
-        <app-question-box class="w-full"></app-question-box>
+        <div class="grid p-3 w-full" style="height: calc(100vh - 80px)">
+            <app-question-box class="col-12 pb-3" />
+            <app-chat-box class="col-12" [ydoc]="ydoc" />
```

Comment:
Not sure how we can handle this but the chatbox is a little too squeezed on my screen. I can't even see the username when the message comes in
![image](https://github.com/user-attachments/assets/39d1667d-1726-494a-9851-637e350180d4)

Reply:
Looks great now!